gsoc/index.html

```diff
@@ -518,7 +522,8 @@ <h4><b>Priyanshu Khandelwal</b></h4>
       <a href="https://github.com/Masquerade0097" class="social-button fab fa-github"></a>
       <p>LINUX FOUNDATION CommunityBridge 2019 - CNCF</p>
     </div>
-    <div class="col-lg-3 centered">
+
```

Comment:
A new row should be created at this line. Check previous examples for how bootstrap works.

Reply:
@vsvipul Added rows to the entire page. Please check